plugins.txt

```diff
@@ -82,7 +82,7 @@ ssh-agent:1.21 # noupdate
 ssh-credentials:1.19
 support-core:2.74
 timestamper:1.13
-token-macro:2.15
+token-macro:266.v44a80cf277fd
```

Comment:
https://github.com/jenkinsci/token-macro-plugin/releases/tag/266.v44a80cf277fd => the version change is because of the "CD" process applied to this process (instead of manual release before). A lot of changes but it's hard to evaluate so let's go \o/